### PR TITLE
Rn entry 2020.07.2 (#1323)

### PR DESCRIPTION
**Pull request by RobertAnderson-okta**
## Description:
- **What's changed?** Added release notes entry for 2020.07.2
- **Is this PR related to a Monolith release?** no

### Resolves:

n/a

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.07.2
+
+| Change                                                      | Expected in Preview Orgs |
+| ----------------------------------------------------------- | ------------------------ |
+| [Bugs fixed in 2020.07.2](#bugs-fixed-in-2020-07-2)         | July 29, 2020            |
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md:9]: nit: bug is singular this week (annoying, I know).

File: packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+### Bugs fixed in 2020.07.2
```
Comment (motikagan-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md:11]: bug singular

File: packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md (hunk 1 above)
Comment (motikagan-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md:9]: keep in mind the anchor link will change too

Comment (RobertAnderson-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md]: Fixed - thanks!

File: packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md (hunk 2 above)
Comment (RobertAnderson-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2020-07-2/index.md]: Fixed

**Pull request merged**